backend/corpora/api_server/logger.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import time
+from logging import LogRecord
+from logging.config import dictConfig
+from typing import List
+
+from pythonjsonlogger import jsonlogger
+from pythonjsonlogger.jsonlogger import merge_record_extra
+
+
+# The fields to log using the json logger.
+LOGGED_FIELDS = ["levelname", "asctime", "name", "message"]
```

Comment:
* `level` might be sufficient
* is `name` useful? it is always `corpora-api-<ENV>`? That seems verbose for the value it provides.

Reply:
we import logging in a few other places so the name of logger would be different. For example, https://github.com/chanzuckerberg/single-cell-data-portal/blob/fa9c0d2fd4b263bd76aa833704e0f53af5b09490/backend/corpora/common/entities/dataset.py#L30

Reply:
I changed the above lines to use the root logger so the log name will be root

Reply:
I find this super useful for tracing where in the code the log messages are coming from

Reply:
we can add [pathname](https://docs.python.org/3/library/logging.html#logrecord-objects) to the log record.

Reply:
will pathname be added?

Reply:
Mostly, I would like to understand how engineers can configure arbitrary "namespaces" for log messages to aid in filtering. If this PR is just going to focus on request logging (and not deal with logging coming from the underlying logic used to generate the response), that seems fine for now, but we should capture ideas for future improvements in a story or the Tech Works Ideas doc.

Reply:
any other loggers that are created using `logging.get()` will use the root logger unless it get configure somewhere else. This means they will all be in JSON format.